### DISCUSSION TITLE
Weird Query

### DISCUSSION THREAD

**Discussion by gabrielpaivadev** [planetscale/discussion#566] [Bug Reports] [answered]
What is it? 

```select ? as `@@socket`, @@max_allowed_packet, @@wait_timeout from dual```

This is consuming 25% of my reading at the database.

I'm using Prisma without serverless adapter

**Comment by orware** [comment, marked as answer]
Hello @gabrielpaivadev,

I know I've responded to your ticket as well yesterday, but that particular query may be originating from Prisma itself as I do see some references to those in the Prisma code when I ran a search:
https://github.com/search?q=org%3Aprisma+%22%40%40socket%22+%22%40%40max_allowed_packet%22+%22%40%40wait_timeout%22&type=code
